python/cugraph/tests/dask/mg_context.py

```diff
@@ -78,17 +79,18 @@ def _prepare_mg(self):
         self._prepare_comms()
 
     def _prepare_cluster(self):
-        self._cluster = CUDACluster(
-            n_workers=self._number_of_devices,
-            rmm_managed_memory=self._rmm_managed_memory
-        )
+        if self._cluster is not None:
```

Comment:
Is there a reason to have _prepare_cluster be a separate thing?  Couldn't this just be done in the `__init__` method?

Reply:
I changed it to your suggestion in the latest commit.